apps/portal/js/wizard.js

```diff
@@ -332,9 +334,21 @@ function getProviderConfig() {
         contentType: "application/json",
         async: false,
         success: function (data) {
-            registerAdvancedProviderUI(data);
             var providerHbs = Handlebars.compile($('#ui-config-hbs').html());
-            $("#provider-config").html(providerHbs(data));
+            if (Object.keys(data[0]).indexOf(groups) > -1) {
+                configGroup = (data[0].grouping);
+                for (i = 0; i < configGroup.length; i++) {
```

Comment:
make sure to declare i, eg: var i . Otherwise this can leak into the global scope.